example-apps/dashrate/src/styles.css

```
@@ -0,0 +1,1021 @@
+@keyframes spin {
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+/* Visually hidden but exposed to screen readers — for labels that are
+   redundant visually (e.g. a placeholder-described field) but still needed
+   for assistive tech. */
+.sr-only {
+  border: 0;
+  clip: rect(0 0 0 0);
```

Stylelint (17.13.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)